repo2docker/buildpacks/r.py

```diff
@@ -69,7 +69,14 @@ def detect(self):
         """
         # If no date is found, then self.checkpoint_date will be False
         # Otherwise, it'll be a date object, which will evaluate to True
-        return bool(self.checkpoint_date)
+        if bool(self.checkpoint_date):
+            return True
+        description_R = 'DESCRIPTION'
+        if not os.path.exists('binder') and os.path.exists(description_R):
+            if not self.checkpoint_date:
+                self._checkpoint_date = datetime.date.today() - datetime.timedelta(days=2)
```

Comment:
could you add a comment as to why "two days before today" is "the right date" to use? Already I can't remember why so I expect in 3months we won't be able to figure this out again :)